apps/dh/api-dh/source/DataHub.WebApi/Modules/Common/Extensions/OperatingIdentityExtensions.cs

```diff
@@ -18,11 +18,11 @@ namespace Energinet.DataHub.WebApi.Modules.Common.Extensions;
 
 public static class OperatingIdentityExtensions
 {
-    public static Guid GetGuid(this IOperatingIdentityDto identity) =>
+    public static string GetSortableString(this IOperatingIdentityDto identity) =>
         identity switch
         {
-            UserIdentityDto user => user.UserId,
-            ActorIdentityDto actor => actor.ActorId,
+            UserIdentityDto user => user.UserId.ToString(),
+            ActorIdentityDto actor => $"{actor.ActorNumber.Value}{actor.ActorRole.Name}",
```

Comment:
```suggestion
            ActorIdentityDto actor => $"{actor.ActorNumber.Value}_{actor.ActorRole.Name}",
```
Would it make sence to seperate the number and role. In case we need to seperate them at some point?

Reply:
This string is just for sorting, it wouldn't matter if there is an `_` in it.